src/nimony/sem.nim

```diff
@@ -8,7 +8,7 @@
 ## Most important task is to turn identifiers into symbols and to perform
 ## type checking.
 
-import std / [tables, sets, syncio, formatfloat, assertions, packedsets, strutils]
+import std / [tables, sets, syncio, formatfloat, assertions, packedsets, strutils, os]
```

Comment:
Don't import os here. Instead move the logic that requires it to `semos.nim`.

Reply:
Sure